Stateless.Tests/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": "1.0.0-*",
+
+  "dependencies": {
+    "NUnit": "3.4.1",
+    "dotnet-test-nunit": "3.4.0-beta-2",
+    "Stateless": "2.5.0-*"
+  },
+
+  "testRunner": "nunit",
+
+  "frameworks": {
```

Comment:
Adding e.g. `net4.6` here as well will make sure the tests are run on both CLRs.